pillar/elastalert.sls

```diff
@@ -173,7 +173,7 @@ elasticsearch:
             - bool:
                 must:
                   - match:
-                      message: AMQPLAIN login refused
+                      message: invalid credentials
```

Comment:
Will this match all cases?

Reply:
The two cases I have comes across in the logs are the following:
`2018-06-07 15:55:46.001 [warning] <0.11298.0> HTTP access denied: user <username>  - invalid credentials`

`AMQPLAIN login refused: user <username>  - invalid credentials`

Based on that, I believe it should match both cases.